diagnostic.md

```diff
@@ -36,13 +41,18 @@ Now, we want to get a **single donut** from the server. Write the `curl` request
 you'd use to retrieve a **single donut**, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request GET http://www.example.com/donuts/id
 ```
 
 Write an AJAX request to retrieve a **single donut** from the server.
 
 ```js
-let getDonut = /* your answer here */;
+let getDonut = function () {
+  return $.ajax({
+    url: 'http://www.example.com/donuts/id',
+    method: 'GET',
+  });
+};
 ```
```

Comment:
Almost! You just need to make sure getDonut passes id for you to be able to use it in url: 

```
let getDonut = function (donutID) {
  return $.ajax({
    url: 'http://www.example.com/donuts/' + donutID,
    method: 'GET',
  });
};
```